src/main/java/ch/njol/skript/conditions/CondItemDespawn.java

```diff
@@ -0,0 +1,34 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import org.bukkit.entity.Item;
+
+@Name("Will Despawn")
+@Description("Checks if the dropped item will be despawned naturally through Minecraft's timer.")
+@Examples({
+	"if all dropped items can despawn naturally:",
+		"\tprevent all dropped items from naturally despawning"
+})
+@Since("INSERT VERSION")
+public class CondItemDespawn extends PropertyCondition<Item> {
+
+	static {
+		PropertyCondition.register(CondItemDespawn.class, PropertyType.WILL, "(despawn naturally|naturally despawn)", "itementities");
+		PropertyCondition.register(CondItemDespawn.class, PropertyType.CAN, "(despawn naturally|naturally despawn)", "itementities");
```

Comment:
Thinking about it a bit more, I'm a little unsure about using CAN for this syntax. Don't remove it though, as I would like to see what others think (it might just be me).

Reply:
CAN seems more accurate to me, since an item may be able to naturally despawn, but it gets picked up first
therefore saying it WILL implies a concrete future outcome

Reply:
i think will is more appropriate here, as this condition is obviously only about natural despawning. both is ok tho